examples/combobox/css/combobox-autocomplete.css

```diff
@@ -58,18 +82,21 @@ ul[role="listbox"] {
 }
 
 ul[role="listbox"] li[role="option"] {
-  border-top: 1px solid transparent;
-  border-bottom: 1px solid transparent;
   display: block;
+  padding-left: 2px;
```

Comment:
The padding in the dropdown was intended to match the padding in the combobox (0.3em on left and right) so that the text would line up nicely.